test/blackbox-tests/test-cases/dialects/good.t/run.t

```diff
@@ -2,6 +2,8 @@ Test the (dialect ...) stanza inside the dune-project file.
 
   $ dune exec ./main.exe
 
+  $ chmod a+w *
```

Comment:
Could you add a comment on these `chmod`s why this is necessary? Someone unaware of that functionality will probably wonder what is going on.

Reply:
This is for the same reason as https://github.com/ocaml/dune/pull/11490#discussion_r1965202171 in `feature.t`. I wonder if there's a better way to do it ?